Modules/RosterManager/Roster.lua

```diff
@@ -571,6 +574,38 @@ CONSTANTS.INVENTORY_TYPES = {
     "INVTYPE_RELIC",
 }
 
+CONSTANTS.INVENTORY_TYPES_SET = UTILS.Set({
```

Comment:
Why not pass the previously defined list instead of repeating?
`UTILS.Set(CONSTANTS.INVENTORY_TYPES)`?

Reply:
Missed that. fixed in next push.